packages/shared/prisma/migrations/20241106122605_add_media_tables/migration.sql

```diff
@@ -0,0 +1,76 @@
+-- CreateTable
+CREATE TABLE "media" (
+    "id" TEXT NOT NULL,
+    "sha_256_hash" CHAR(44) NOT NULL,
+    "project_id" TEXT NOT NULL,
+    "created_at" TIMESTAMP(3) NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    "updated_at" TIMESTAMP(3) NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    "expires_at" TIMESTAMP(3),
```

Comment:
Remove expiresAt and deletedAt while it is not used yet. Only add when cleanup job is added